### PR TITLE
Add Github issue template

### PR DESCRIPTION
**Pull request by dudanogueira**
### What's being changed:

Adds Github Issue template. See it in action here:
https://github.com/dudanogueira/weaviate/issues/new/choose

### Review checklist

- [ ] Documentation has been updated, if necessary. Link to changed documentation:
- [ ] Chaos pipeline run or not necessary. Link to pipeline:
- [ ] All new code is covered by tests where it is reasonable.
- [ ] Performance tests have been run or not necessary.

### CODE REVIEW

**Comment by etiennedi** [comment]
Awesome! Thanks, @dudanogueira.

One thought: On the bug template, many critical questions are all combined in the same field. That makes it relatively easy for the user to ignore one of the steps. What if we split the following into explicit text boxes:
* How to reproduce
* Expected Behavior
* Actual Behavior
* Version

Ideally, every bug ticket should look something like the following: https://github.com/weaviate/weaviate/issues/3762

Wdyt?

**Comment by etiennedi** [comment]
On the feature request page:

image

What if we made the first link ("requested this feature") point [here](https://weaviate.io/developers/weaviate/roadmap) instead?

**Comment by dudanogueira** [comment]
Nice inputs!! THanks!
It was updated.
preview here:  https://github.com/dudanogueira/weaviate/issues/new/choose

File: .github/ISSUE_TEMPLATE/create_issue.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+name: Found a bug?
+description: Report your bug here.
+labels: ["bug", "triage"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        #### Before you get started
+        * Check that sure someone hasn't already opened a similar [issue](https://github.com/weaviate/weaviate-io/issues).
```
Comment (parkerduckworth) [inline .github/ISSUE_TEMPLATE/create_issue.yml:10]: Maybe change `Check that sure someone [...]` to `Check to make sure that someone [...]`

**Review by parkerduckworth — COMMENTED** [review]
One small suggestion, otherwise looks great!

File: .github/ISSUE_TEMPLATE/create_issue.yml (hunk 1 above)
Comment (sebawita) [reply .github/ISSUE_TEMPLATE/create_issue.yml]: done

**Review by parkerduckworth — APPROVED** [review]

**Pull request merged**